### PR TITLE
Editor: do not watch and refresh blocks when there are not ready to SDPA-433

### PR DESCRIPTION
**Pull request by vied12**

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: scripts/superdesk/editor2/add-content.js [hunk 1]
```diff
@@ -19,11 +19,15 @@ function($window) {
...
             // listen for update state signals
-            scope.$parent.$on('sdAddContent::updateState', function(signal, event, editorElem) {
+            var listener = scope.$parent.$on('sdAddContent::updateState', function(signal, event, editorElem) {
                 vm.updateState(event, editorElem);
             });
             // update on resize
             angular.element($window).on('resize', vm.updateState);
+            scope.$on('$destroy', function() {
+                angular.element($window).off('resize', vm.updateState);
+                listener();
```
Comment (actionless) [inline scripts/superdesk/editor2/add-content.js]: what happens when you are calling it?

Comment (vied12) [inline scripts/superdesk/editor2/add-content.js]: it unbinds the listener

Comment (actionless) [inline scripts/superdesk/editor2/add-content.js]: i think it should be stated a bit more explicitly what it's a de-registration function, mb rename it to `unbindListener`

**Comment by actionless** [comment]
LGTM

**Comment by actionless** [comment]
LGTM

**Pull request merged**